src/Components/Atoms/FATextField/index.tsx

```diff
@@ -33,25 +23,17 @@ export const FATextField = memo((props: IInput) => {
     adornment && adornmentPosition === "end" ? AddAdornment() : null;
 
   return (
-    <FormControl data-testid={testID}>
-      <TextField
-        {...materialUIProps}
-        disabled={disabled}
-        type={type}
-        label={label}
-        variant={variants}
-        onChange={onChange}
-        value={value}
-        InputProps={{
-          startAdornment: startAdornment,
-          endAdornment: endAdornment,
-        }}
-        aria-describedby="component-error-text"
-      />
-      <FormHelperText id="component-error-text" error>
-        {helperText}
-      </FormHelperText>
-    </FormControl>
+    <TextField
+      data-testid={testID}
+      name={name}
+      variant={variants}
+      {...materialUIProps}
+      InputProps={{
+        startAdornment: startAdornment,
+        endAdornment: endAdornment,
+      }}
+      aria-describedby="component-error-text"
```

Comment:
if we want to use `aria-describedby` attribute, better to using dynamic value

Reply:
working fine without it